### PR TITLE
updates to how it reports the named server id to sync up with changes…

### PR DESCRIPTION
**Pull request by fillup**
… in admin api

### CODE REVIEW

File: agent.go [hunk 1]
```diff
@@ -69,7 +70,7 @@ type TaskLogEntry struct {
 	Timestamp         int64   `json:"Timestamp"`
 	EntryType         string  `json:"EntryType"`
 	ServerCountry     string  `json:"ServerCountry,omitempty"`
-	ServerID          string  `json:"ServerID,omitempty"`
+	NamedServerID     uint    `json:"NamedServerID,omitempty"`
```
Comment (Baggerone) [inline agent.go]: Didn't we want the ServerHost here?

File: lib/logqueue/logqueue_test.go [hunk 2]
```diff
@@ -47,10 +47,10 @@ func TestManager(t *testing.T) {
 
 	testLogs := []agent.TaskLogEntry{
 		{
-			EntryType: agent.TypePing,
-			Latency:   12.123,
-			Timestamp: 1525877951,
-			ServerID:  "1234",
+			EntryType:     agent.TypePing,
+			Latency:       12.123,
+			Timestamp:     1525877951,
```
Comment (Baggerone) [inline lib/logqueue/logqueue_test.go]: Add ServerHost?

File: lib/speedtestnet/serverlist.go [hunk 3]
```diff
@@ -16,1174 +16,1174 @@ func getServerList() []server {
...
-			Name: "Jersey City, NJ",
-			Country: "United States",
-			CC: "US",
-			Sponsor: "QTS Data Centers",
-			ID: "14854",
-			Host: "jcy1sp1.qtsdatacenters.com:8080",
+			Name:      "Jersey City, NJ",
+			Country:   "United States",
+			CC:        "US",
+			Sponsor:   "QTS TaskData Centers",
```
Comment (Baggerone) [inline lib/speedtestnet/serverlist.go]: Lol. The Refactor-Rename must have changed this one too.

File: lib/tasks/tasks.go [hunk 4]
```diff
@@ -62,8 +62,7 @@ func UpdateTasks(
 						logEntry := agent.GetTaskLogEntry(agent.TypePing)
 						logEntry.Latency = spTestResults.Latency.Seconds() * 1000
 						logEntry.PacketLossPercent = spTestResults.PacketLossPercent
-						logEntry.ServerCountry = task.NamedServer.Country.Code
-						logEntry.ServerID = task.NamedServer.SpeedTestNetServerID
+						logEntry.NamedServerID = task.NamedServerID
```
Comment (Baggerone) [inline lib/tasks/tasks.go]: Add ServerHost?

File: lib/tasks/tasks.go [hunk 5]
```diff
@@ -77,15 +76,14 @@ func UpdateTasks(
...
+					spTestResults, err := spdTestRunner.Run(task.TaskData)
 					if err != nil {
 						logError("1525291938", "Error running speed test: ", err, newLogs)
 					} else {
 						logEntry := agent.GetTaskLogEntry(agent.TypeSpeedTest)
 						logEntry.Download = spTestResults.Download
 						logEntry.Upload = spTestResults.Upload
-						logEntry.ServerCountry = task.NamedServer.Country.Code
-						logEntry.ServerID = task.NamedServer.SpeedTestNetServerID
+						logEntry.NamedServerID = task.NamedServerID
```
Comment (Baggerone) [inline lib/tasks/tasks.go]: Add ServerHost?

File: agent.go (hunk 1 above)
Comment (fillup) [reply agent.go]: The API loads the NamedServer and populates additional fields on the log like `ServerCountry`, `ServerCoordinates`, `ServerName`, and `ServerHost`

File: lib/logqueue/logqueue_test.go (hunk 2 above)
Comment (fillup) [reply lib/logqueue/logqueue_test.go]: Same reason as above

File: lib/speedtestnet/serverlist.go (hunk 3 above)
Comment (fillup) [reply lib/speedtestnet/serverlist.go]: :-)

File: lib/tasks/tasks.go (hunk 4 above)
Comment (fillup) [reply lib/tasks/tasks.go]: Same as above

File: lib/tasks/tasks.go (hunk 5 above)
Comment (fillup) [reply lib/tasks/tasks.go]: Same as above

**Pull request merged**